lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -123,45 +117,45 @@ export default class LockMonitor {
   }
 
   private async handlePeriodicPolling (): Promise<void> {
+    this.currentLockState = await this.getCurrentLockState();

```

Comment:
Calling out that internally `getCurrentLockState()` will "help" broadcast the lock transaction if it is not found, so the `LockMonitor` is not in "pure read mode" even if lock update is disabled. But I don't think this is a big deal. If you have concern for this behavior, let me know!

Reply:
I think it's fine as long as it doesn't update the lock internally. Broadcasting just lets other bitcoin nodes know it exists, and it shouldn't even happen often if we don't create new locks.

Reply:
Same here. Broadcast is not the same as 'updating'. Someone already updated it, and this node is just broadcasting it which is a noop if already broadcasted.